src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -605,6 +605,15 @@
 	IEmoji getEmojiByName(String name);
 
 	/**
+	 * This adds a custom emoji to the guild.
+	 *
+	 * @param name The name, <b>without colons</b>.
+	 * @param image The BufferedImage of the Emoji.
+	 * @return The emoji.
```

Comment:
I have no idea where I was looking then. I added `DiscordException `and `MissingPermissionsException`, but I could not find for sure the code that would throw a `RateLimitException`, but I would think that any discord call could return a rate limit hit?